content/docs/user-guide/how-to/add-deps-or-outs-to-a-stage.md

```diff
@@ -0,0 +1,55 @@
+# How to Add Dependencies or Outputs
+
+We have executed a stage, but later notice that some of the files/directories it
```

Comment:
Is it needed? the whole paragraph?

Reply:
It's a single sentence on motivation.

Like I mentioned, I realized that this how-to is really only useful for people who have already reproed their dvc.yaml or used run from the beginning. For people that write dvc.yaml manually and haven't reproed it, it's obvious how to add more deps/outs.

Reply:
> this how-to is really only useful for people who have already reproed their dvc.yaml or used run from the beginning

Let me think this through thouogh... ⌛

Reply:
OK I went your way on this. PTAL